python/py-folium/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-folium
+version             0.14.0
+
+categories-append   devel graphics
+platforms           {darwin any}
+supported_archs     noarch
+license             MIT
+maintainers         nomaintainer
+
+description         Generate complex HTML+JS pages with Python
+
+long_description    {*}${description}. This library is a spinoff from \
+                    folium, that would host the non-map-specific features.
+
+homepage            https://github.com/python-visualization/folium/
+
+checksums           rmd160  bc1bf63bbe263cc9e66d6d93b33c67d6ca68e55a \
+                    sha256  8ec44697d18f5932e0fdaee8b19da98625de4d0e72cb30ef56f9479f18e11b9f \
+                    size    93500
+
+python.versions     38 39 310 311
+python.pep517       yes
+
+if {${subport} ne ${name}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
```

Comment:
This shouldn't be needed as it is included automatically with the python.pep517 where the default backend is setuptools. 

However, one of the ports is missing a dependency on setuptools_scm as you can see from the CI.

Reply:
Thank you very much. Fixed, I hope.